.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   node10:
     docker:
-      - image: circleci/node:10
+      - image: circleci/node:15
```

Comment:
I don't think this is a good idea :-)

Reply:
Already reverted, I just wanted to see what would happen.